frontend/src/scenes/dashboard/DeleteDashboardModal.tsx

```diff
@@ -0,0 +1,66 @@
+import { useActions, useValues } from 'kea'
+
+import { LemonButton } from 'lib/components/LemonButton'
+
+import { LemonModal } from 'lib/components/LemonModal'
+import { Form } from 'kea-forms'
+import { deleteDashboardLogic } from 'scenes/dashboard/deleteDashboardLogic'
+import { DashboardType } from '~/types'
+import { Field } from 'lib/forms/Field'
+import { LemonCheckbox } from 'lib/components/LemonCheckbox'
+
+export function DeleteDashboardModal({ dashboard }: { dashboard?: DashboardType }): JSX.Element {
```

Comment:
We don't know which dashboard it is when we add the modal to the list page... This is only to add the name to the modal so I bet I can remove this prop entirely

Reply:
Hmm, now I see that we shouldn't be using a prop here, since state should come from `deleteDashboardLogic` (as a single source of truth). But it should be very straightforward to have `dashboardAboutToBeDeleted` as a `deleteDashboardLogic` reducer, shouldn't it?